app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,43 @@
+
+
+require_relative 'stats_helpers/donations.rb'
+require_relative 'stats_helpers/stats_html.rb'
+
+class StatsController < ApplicationController
+    def donation_totals
+        @result = ActiveRecord::Base.connection.execute($donation_query)
```

Comment:
nit: you can just make this whole method:

def donation_totals
        ActiveRecord::Base.connection.execute($donation_query)
end